src/components/DatePicker/DatePicker.js

```diff
@@ -60,14 +62,21 @@ export default class DatePicker extends Component {
             ? [new rangePlugin({ input: this.toInputField })]
             : '',
         clickOpens: true,
-        onChange: () => {
-          this.props.onChange();
-        },
+        onChange: this.props.onChange,
         onReady: (selectedDates, dateStr, instance) => {
           this.updateClassNames(instance);
         },
         nextArrow: this.rightArrowHTML(),
         leftArrow: this.leftArrowHTML(),
+        onMonthChange: (electedDates, dateStr, instance) => {
```

Comment:
Are these all the events that we should be exposing to `updateClassNames`? Or are these the only ones developers would care about?

Reply:
Good catch. There was another case that I didn't catch: `onValueChanged`. Updated to reflect catching that event.